plugins/Morpheus/javascripts/piwikHelper.js

```diff
@@ -13,6 +13,8 @@ function _pk_translate(translationStringId, values) {
         if (typeof values != 'undefined' && values && values.length) {
             values.unshift(translation);
             return sprintf.apply(null, values);
+        } else if (values && values.length === 0) {
```

Comment:
Guess simply using an else would be easier and should also work in all cases

Reply:
Updated as suggested